src/redux/createReducer.ts

```diff
@@ -94,11 +95,13 @@ function handleChangeQueueSemaphore(
   };
 }
 
-export default (comparisonFn: Function = getSimilarActionInQueue) => (
-  state: NetworkState = {
-    ...initialState,
-  },
-  action: *,
+type ComparisonFn = (
```

Comment:
Could we move this type definition to the top? It's also used by the function above.